CONTRIBUTING.md

```diff
@@ -2,31 +2,41 @@
 
 Jest to drugi najważniejszy plik po README w projekcie. Dotyczy on osób, które chcą w sposób aktywny wpłynąć na rozwój naszego forum.
 
-# Tworzenie `issue`
+## Standardy kodu
```

Comment:
IMHO wyłącznie opisywanie standardu formatowania w jakimś pliku README jest równoznaczne z tym, że te standardy nie istnieją. Jeżeli chcecie takowe wprowadzić, konieczne jest wymuszenie tego poprzez linter, bo inaczej to tak jak pisanie testów jednostkowych bez continuous integration - wtedy testów tak na prawdę nie ma.

Przykładowo do Javascript'u, skoro chcecie przestrzegać ES6, będzie to eslint (https://eslint.org/). Jest pewnie sporo kodu legacy, więc możecie początkowo ograniczyć liczbę plików i stopniowo to zwiększać + poprawy takie jak wcięcia i braki średników eslint jest w stanie naprawić sam, więc zrobicie to automatycznie dla całego codebase'u.

Reply:
No spoko, tylko jak włączymy coś pod PSR-2 to błędy będą w dosłownie każdej linii kodu po kolei. I poprawienie tego nie będzie łatwe, praktycznie trzeba by przepisać cały kod od nowa (nazwy klas, metod, funkcji, zmiennych) i refaktorować każdy jeden plik. Moim zdaniem określenie standardu tutaj ma sens, będziemy po prostu zwracać na to uwagę przy code review.

Reply:
Standardy formatowania spoko. Jak chcemy to rozwiązać bez narzędzi automatyzujących? Nie wrzucę pliku ustawień phpstorma bo ktoś może mieć inny edytor. Trzeba byłoby podpiąć pod repozytorium narzędzie, które będzie każdy commit monitorować.

Reply:
Korzystałem nie raz ze StyleCI do PSR-2, jest spoko i można to łatwo podpiąć. Ale co nam po tym, gdy calutki kod oznaczy jako do poprawy? Ma to jakiś sens, aby widzieć co jest w danej zmianie nie tak. Ale patrząc na to, że będzie pokazywał wszystko jako złe, nie widzę sensu podpięcia tego dopóki kod nie będzie chociaż jako tako ogarnięty.